panther_driver/src/driver_node.py

```diff
@@ -64,7 +64,7 @@ def get_ADC_measurement(name: str, config_file):
     data = config_file[name]
     path = data["path"]
     raw_value = read_file(path)
-    value = raw_value * data["LSB"]
+    value = (raw_value-data["offset"]) * data["LSB"]
```

Comment:
```suggestion
    value = (raw_value - data["offset"]) * data["LSB"]
```
More in terms of aesthetic, but I am more used to spaces if there is **-** sign.

Reply:
:+1: